content/pages/privacy-policy.rst

```diff
@@ -105,7 +105,7 @@ Why do we collect this information?
 - We limit our use of your User Personal Information to the purposes listed in this Privacy Statement.
   If we need to use your User Personal Information for other purposes, we will ask your permission first.
   You can always see what information we have in your
-  `user account <https://readthedocs.org/accounts/edit/>`__.
+  `user account <https://app.readthedocs.org/accounts/edit/>`__.
```

Comment:
I think most of these probably need some similar solution to the dashboard links, where we have a way to link to a page that asks the user to go to .com or .org. It's probably out of scope for this PR though 🙃